### PR TITLE
Added GIT flow

### PR DESCRIPTION
**Pull request by seansan**
If not here, then please add it to http://openmage.github.io/magento-lts/install.html ... it is good to have it. This way if he flow changes or can be improved people can add to it,

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,12 +21,27 @@ Note, the branches older than `1.9.3.x` that were created before this strategy c
...
 
+### Using Composer
 Download the latest archive and extract it, clone the repo, or add a composer dependency to your existing project like so:
 
 ```json
 "openmage/magento-lts": "1.9.4.x"
 ```
 
+### Using Git
+Go to `https://github.com/OpenMage/magento-lts` and fork the project. Enter your web directory and start a new fit project utilizing `git init`. Set git remote to your forked repo using `git remote add origin https://github.com/<YOUR GIT USERNAME>/magento-lts`. Issue `git pull origin master`. Then add the official LTS repo as an upstream using `git remote add upstream https://github.com/OpenMage/magento-lts`. Issue `git pull upstream v19.4.4` or the latest version you want to pull. Be sure to check for conflicts before merging all in the next step. Dont forget to git commit and add files to your own forked repo.
```
Comment (kiatng) [inline README.md:32]: Typo _fit_ -> _git_

**Comment by colinmollenhour** [comment]
@julienloizelet Currently our HEAD branch is `1.9.4.x` instead of the common;y used `master`. To avoid having to know the name you can also do `git pull origin/HEAD`.

**Comment by julienloizelet** [comment]
> @julienloizelet Currently our HEAD branch is `1.9.4.x` instead of the common;y used `master`. To avoid having to know the name you can also do `git pull origin/HEAD`.

Thanks. So maybe  `git pull origin master` should be changed to `git pull origin/HEAD` or `git pull origin 1.9.4.x`

**Comment by seansan** [comment]
Can someone review? Or is it ok? 

(we can always make further improvements later anyways)

**Comment by sreichel** [comment]
@all-contributors please add seansan for doc

**Comment by sreichel** [comment]
@all-contributors please add @seansan for doc

**Pull request merged**